### PR TITLE
Fixed FixFeedModelSpec

### LINKED ISSUE #861: Fix FixFeedModelSpec
Something I didn't get during my review.

```        
actualModel.calculateActivePower(
          ModelState.ConstantState,
          CalcRelevantData.FixedRelevantData,
        ) =~ expectedPower
```

### PR DESCRIPTION
**Pull request by danielfeismann**
resolves #861

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Not stopping correctly on failed power flow if configured  to stop [#800](https://github.com/ie3-institute/simona/issues/800)
 - Finally fixing `RuntimeEventListenerSpec` [#849](https://github.com/ie3-institute/simona/issues/849)
 - Fixed result output for thermal houses and cylindrical storages [#844](https://github.com/ie3-institute/simona/issues/844)
+- Fixed FixFeedModelSpec [#861](https://github.com/ie3-institute/simona/issues/861)
```
Comment (pierrepetersmeier) [inline CHANGELOG.md:83]:
```suggestion
- Fixed FixedFeedModelSpec [#861](https://github.com/ie3-institute/simona/issues/861)
```

**Review by pierrepetersmeier — COMMENTED** [review]
LGTM ;)

File: CHANGELOG.md (hunk 1 above)
Comment (danielfeismann) [reply CHANGELOG.md]: Good catch!

**Review by pierrepetersmeier — COMMENTED** [review]
LGTM

**Review by jo-bao — APPROVED** [review]
Well done ;-)

**Pull request merged**